src/xbot/common/properties/RobotDatabaseStorage.java

```diff
@@ -2,7 +2,7 @@
 
 public class RobotDatabaseStorage extends DatabaseStorageBase {
 
-    private static String location = "/TeamDatabase";
+    private static String location = "/home/lvuser/TeamDatabase";
```

Comment:
- Good call on final
- I think hardcoding here is OK - that's sort of the point of this injected class.